code/game/turfs/open.dm

```diff
@@ -63,7 +64,16 @@
 				return 0
 			if(C.m_intent=="walk" && (lube&NO_SLIP_WHEN_WALKING))
 				return 0
-
+		if(istype(C, /mob/living/carbon/human))
```

Comment:
hmm, is it worth defining a mob proc process_walk and moving some of the checks into that?

This is a pretty deeply nested if chain